docs/getting_started/getting_started.rst

```diff
@@ -24,24 +24,20 @@ order:
     #. Make a new Github account on https://www.github.com if you don't already have one.
     #. Fork the oh-mainline Github repository located here at https://github.com/openhatch/oh-mainline. Click on the fork button located on the upper  right corner of the project page. Now you have your own personal copy of the oh-mainline repository.
     #. Install git the version control system. If you have already done so, skip to the next step.
-    #. Clone your personal copy of the oh-mainline repository to your computer by typing this command into your terminal::
-
-        $ git clone https://github.com/<YOUR_GITHUB_USERNAME>/oh-mainline.git
+    #. Clone your personal copy of the oh-mainline repository to your computer by typing this command into your terminal
+    
```

Comment:
The Github web view points out that there's unnecessary whitespace at the end of this line.